core/src/auth_service/auth_service.bal

```diff
@@ -0,0 +1,67 @@
+import ballerina/config as conf;
+import ballerina/http;
+import ballerina/jwt;
+
+
+jwt:InboundJwtAuthProvider jwtAuthProvider = new ({
+    issuer: "environment platform",
+    trustStoreConfig: {
+        certificateAlias: "ballerina",
```

Comment:
We have the certificate with the alias `ballerina` in the keystore right?

Reply:
yes.